File: 백준/Silver/4134. 다음 소수/다음 소수.py
import math

def getPrime(n):
    while(1):
        if n < 2 :
            return 2
        is_prime = True
        for i in range(2, int(math.sqrt(n)) + 1):
            if n % i == 0: 
                is_prime = False
                break
        if is_prime:
            return n
        n += 1
        
num = int(input())
while(num > 0):
    number = int(input())
    print(getPrime(number))
    num -= 1

File: 백준/Silver/4134. 다음 소수/다음 소수.c
#include <stdio.h>
#include <math.h>
#include <stdbool.h>

long long getPrime(long long n);

int main(void) {
	int n;
	long long num;
	scanf("%d", &n);
	while (n > 0) {
		scanf("%lld", &num);
		printf("%lld\n", getPrime(num));
		n--;
	}

	return 0;
}

long long getPrime(long long n) {
	if (n < 2) return 2;
	while (1) {
		bool prime = true;
		for (int i = 2; i < (long long)sqrt(n) + 1; i++) {
			if (n % i == 0) {
				prime = false;
				break;
			}
		}
		if (prime) return n;
		n++;
	}
}